main/usbutils/template.py

```diff
@@ -1,6 +1,6 @@
 pkgname = "usbutils"
 pkgver = "017"
-pkgrel = 1
+pkgrel = 2
```

Comment:
@q66 is the pkgrel bumping necessary? or will packages automatically build `usbutils-devel` when needed?

if the pkgrel bump is necessary, then should all other packages that depend on usbutils be relbumped too?

Reply:
> is the pkgrel bumping necessary?   

yes

> then should all other packages that depend on usbutils be relbumped too?  

no. they don't use usbutils-devel (and wouldn't even install it anyway as they don't depend on usbutils-devel, no difference in rebuild)